src/NUnitFramework/framework/Constraints/SubstringConstraint.cs

```diff
@@ -57,7 +57,7 @@ protected override bool Matches(string? actual)
         /// than <paramref name="comparisonType"/> was already set.</exception>
         public SubstringConstraint Using(StringComparison comparisonType)
         {
-            if (_comparisonType == null)
+            if (_comparisonType is null)
```

Comment:
Flagged by updating `CSharpIsNullAnalyzer`. `_comparisonType` is a nullable enum which must now be detectable